### PR TITLE
FormHelper sourcesValues feature docs

### PR DESCRIPTION
**Pull request by inoas**
Ref: https://github.com/cakephp/cakephp/pull/9127

### CODE REVIEW

**Comment by inoas** [comment]
Is this GTM?

File: en/views/helpers/form.rst [hunk 1]
```diff
@@ -98,6 +98,52 @@ The ``$options`` array is where most of the form configuration
...
+
+The default values source for a FormHelper is ``context``. Thus the values will be fetched from the Form's context.
+The default contexts, such as ``EntityContext`` or ``NullContext``, will fetch their values from ``$request->data``,
+which is being autofilled with ``HTTP`` ``POST``/``PUT``/``PATCH`` body data. If there is no such data for a given field,
+but an Entity is given, the value will be fetched from the Entity instead.
+
+You can change this behavior by specifying ``valuesSources`` either as an option or by calling it as a setter.
+The default is ``['context']``.
+
+The `valuesSources` can be set to ``context``, ``data`` and ``query``. You can either set the Form to use one source
```
Comment (markstory) [inline en/views/helpers/form.rst]: backticks should be doubled. `valuesSources` has only single backticks.

**Comment by markstory** [comment]
I've retargetted this to 3.4, and will fix up the additional commits being added when I merge this.

**Comment by markstory** [comment]
Merged in ca1e900a0b5f144833d86e36811444b122ac220c

**Pull request closed without merging**